src/static/js/modules/nav-primary.js

```diff
@@ -28,16 +37,26 @@ function init() {
     var $this = $( this );
     var $thisSubNav = $this.siblings( '.js-sub-nav' );
     var $otherSubNavs = $subNavs.not( $thisSubNav );
+    var $firstLink = $thisSubNav.find( 'a' )[0];
```

Comment:
Agree with @sebworks. `$` prefix should only designate a variable contains a jQuery object, to avoid confusing situations like this where `$obj.focus()` and `obj.focus()` are using two different focus methods (jQuery's and native, respectively).